src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: $container-width;
+}
+
+.card-wrapper {
+  margin-right: 48px;
+
+  &:nth-child(-n + 4) {
+    margin-bottom: 46px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: auto;
+    margin-bottom: 40px;
+    align-self: center  ;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rate-text {
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+    align-self: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-text-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__btn-buy {
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    text-align: center;
+    line-height: 16px;
+    padding: 12px 0;
+    border: 1px solid $accent-color;
+    border-radius: 5px;
+    background-color: $accent-color;
+    color: #fff;
+  }
+
+  &__btn-buy:hover {
+    background: #fff;
+    color: $accent-color;
+    transition: 0.3s color;
```

Comment:
move transition to btn styles

Reply:
It is haven't been fixed + it doesn't work